app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/labtest/LabTestDetailFragment.kt

```diff
@@ -42,7 +42,9 @@ class LabTestDetailFragment : Fragment(R.layout.fragment_lab_test_detail) {
     private fun setToolBar() {
         binding.toolbar.apply {
             ivLeftOption.visibility = View.VISIBLE
+            ivRightOption.visibility = View.VISIBLE
```

Comment:
Download option is visible based on isReportAvailable response from API endpoint. So please removed newly added lines here. This implementation will be covered in separate PR.